data_pmt.go

```diff
@@ -8,12 +8,29 @@ import (
 
 // Stream types
 const (
+	StreamTypeMPEG1Video                 = 0x01
```

Comment:
I've considered making this its own type. That would allow to implement Stringer for it. But that would break back compatibility. What do you think?

Reply:
I like the idea of creating its own type

Reply:
you got it!

btw looks like we don't even need `PSISectionHeader.TableType` anymore